api/src/main/java/org/openmrs/module/registrationapp/model/DropdownWidget.java

```diff
@@ -33,6 +33,9 @@ public void setConfig(Config config) {
         @JsonProperty
         private Boolean hideEmptyLabel;
 
+        @JsonProperty
+        private Boolean autocomplete;
```

Comment:
The property name should match the name of the getter and setter.  So either call this "autocomplete" or "autocompleteEnabled" but not both.